config/env/production.js

```diff
@@ -7,6 +7,7 @@ module.exports = {
     certificate: './config/sslcerts/cert.pem'
   },
   port: process.env.PORT || 8443,
+  host: process.env.HOST || '127.0.0.1',
```

Comment:
@farajfarook we should be consistent and set the production config to 0.0.0.0 too.
IMO it's not *that* much of a security risk because users usually clone on their own machine and set it up later to run on production usually using build tools.

More than that, nowadays people use docker (or other types of VMs) and it is required to bind to a non-loopback host/ip to make the app accessible.